packages/lib/server/repository/google.ts

```diff
@@ -1,48 +0,0 @@
-import type { Prisma } from "@prisma/client";
-import type { Credentials } from "google-auth-library";
-
-import { CredentialRepository } from "./credential";
-import { SelectedCalendarRepository } from "./selectedCalendar";
-
-export class GoogleRepository {
```

Comment:
this is renamed to `GoogleService` and moved to `/service` folder. All methods here are copied into the new class.